cmd/tuples/read.go

```diff
@@ -19,80 +19,88 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"os"
 
 	cmdutils "github.com/openfga/cli/lib/cmd-utils"
 	openfga "github.com/openfga/go-sdk"
 	"github.com/openfga/go-sdk/client"
 	"github.com/spf13/cobra"
 )
 
-// MaxReadPagesLength Limit the tuples so that we are not paginating indefinitely.
-var MaxReadPagesLength = 20
```

Comment:
Can you keep this? Update the init so we are referencing this instead of `MaxReadChangesPagesLength`. That was a typo.